apps/extension/src/content-scripts/injected-penumbra-global.ts

```diff
@@ -57,6 +57,7 @@ class PraxInjection {
   private stateEvents = new EventTarget();
 
   private injection: Readonly<PenumbraProvider> = Object.freeze({
+    request: () => this.postConnectRequest().then(() => Promise.resolve()),
```

Comment:
issue (blocking): can you add code documentation on the fact that this is temporary and should be removed after a pd point release?

Reply:
commented

Reply:
this was approved by another review, but you've left a blocking comment, so i will avoid merging this unless you re-review.

go ahead and merge this if you intend to include it in a release.